disperser/apiserver/server.go

```diff
@@ -169,6 +170,7 @@ func (s *DispersalServer) DisperseBlobAuthenticated(stream pb.Disperser_Disperse
 }
 
 func (s *DispersalServer) DisperseBlob(ctx context.Context, req *pb.DisperseBlobRequest) (*pb.DisperseBlobReply, error) {
+	s.metrics.IncrementBlobRequestNum("DisperseBlob")
```

Comment:
I'm wondering if it would be more useful to emit this at the end instead of the beginning. Because right now IncrementBlobRequestNum has status code = total. Total should equal "success + failures" anyways.

Reply:
Having a time series for "total" will make the monitoring query easy, just `1 - 500 / total`, and also more stable (no need to update when there is a new failure case to add here).

We currently have 4 failure codes; adding more will make the query itself more complex if "total" has to be expanded